packages/browser-utils/src/metrics/inp.ts

```diff
@@ -84,6 +84,9 @@ function _trackINP(): () => void {
     const rootSpan = activeSpan ? getRootSpan(activeSpan) : undefined;
 
     const routeName = rootSpan ? spanToJSON(rootSpan).description : undefined;
+    if (routeName === undefined) {
```

Comment:
```suggestion
    if (!routeName) {
```

I think this is good enough and slightly smaller!